pkg/utils/build/build.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2019 The MayaData Authors.
+Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
Is this intentional? :thinking:

Reply:
does this needs the fix @AmitKumarDas ? in other files also

Reply:
Yes... Parts of the file were from `https://github.com/uber/makisu`

Reply:
> This is required. Parts of the file were from https://github.com/uber/makisu

Nice, I am in favor of keeping it. :hugs: